setup.py

```diff
@@ -67,6 +67,8 @@ def run_tests(self):
             "scriptworker = scriptworker.worker:main",
             "create_initial_gpg_homedirs = scriptworker.gpg:create_initial_gpg_homedirs",
             "rebuild_gpg_homedirs = scriptworker.gpg:rebuild_gpg_homedirs",
+            "nagios_check_pending_tasks = scriptworker.monitoring.nagios_pending_tasks:query_pending_task_count",
+            "nagios_check_file_ages = scriptworker.monitoring.nagios_file_age_check:run_file_age_checks",
```

Comment:
This is awesome.
However, we're adding a new subdirectory, and by default we're going to be missing the scriptworker/monitoring directory in the pypi package.

We need to add `scriptworker.monitoring` to the [`packages` list](https://github.com/mozilla-releng/scriptworker/blob/3269a027c20e27a6fd5efffd20310510a1f7101f/setup.py#L62)

To test, `python setup.py sdist` . You'll get a `dist/scriptworker-VERSION.tar.gz` that you can extract or `tar tvf` to view the contents.  Make sure `scriptworker/monitoring/*.py` are included.  You probably need to touch and `git add` a `scriptworker/monitoring/__init__.py` for that to work.